src/ui/public/agg_response/tabify/_response_writer.js

```diff
@@ -3,8 +3,9 @@ import AggConfigResult from 'ui/vis/agg_config_result';
 import { TabifyTable } from 'ui/agg_response/tabify/_table';
 import { TabifyTableGroup } from 'ui/agg_response/tabify/_table_group';
 import { tabifyGetColumns } from 'ui/agg_response/tabify/_get_columns';
+import { createLegacyClass } from '../../utils/legacy_class';
```

Comment:
@epixa Why don't we do relative paths?

Reply:
@archanid I don't understand the question - this is using relative paths.

Reply:
@archanid There's a "magic" `ui/` prefix available in Webpack, so plugins etc can import stuff that lives in the `src/ui/public` folder in a simple way. We're gradually getting rid of it (through packages, the new plugin system, etc), but it's still used in tons of places. Here I just changed the stuff within the `src/ui/public` folder to do relative requires instead of using the magic `ui/` prefix.

Reply:
Ah, interesting. Yeah, I see that `ui/` prefix and it confuses me. Relative paths are what we use in other places. It's interesting to see how things evolved. First we were probably using relative paths, but saw that `../../ui/` was so prevalent, and we wanted an "absolute" shortcut to it. Now we're more comfortable with relative paths again.

Reply:
So is the plan to use relative paths even when _not within_ `ui`?